node/resolvers/items.ts

```diff
@@ -194,10 +197,13 @@ export const mutations = {
       })
     }
 
-    const newOrderForm = await clients.checkout.updateItems(
+    const newOrderForm = await clients.checkout.addItem(
       orderFormId!,
       cleanItems,
-      splitItem
+      {
+        allowOutdatedData,
+        splitItem: splitItem
```

Comment:
You can leave just `splitItem`, as `{ splitItem: splitItem }` of syntatic sugar of `{ splitItem }`